src/stores/entities/specs.go

```diff
@@ -0,0 +1,23 @@
+package entities
+
+import "time"
+
+// HashicorpSpecs is the specs format for a Hashicorp Vault client
+// TODO: Remove annotations when type from service layer is used
+type HashicorpSpecs struct {
```

Comment:
Since you extracted this Specs, which I believe it is a good decision, I would expect to see the same for AKV and AWS, or at least have a ticket to do so in the backlog

Reply:
https://app.zenhub.com/workspaces/orchestrate-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/169